java/src/jmri/jmrit/operations/automation/actions/Action.java

```diff
@@ -173,7 +174,7 @@ public int finishAction(boolean success, Object[] buttons) {
      *                buttons, the second button becomes the default
      * @param success true if action succeeded
      * @param message the text to be displayed
-     * @return which button was pressed, NO_MESSAGE_SENT, CLOSED
+     * @return array number for which button was pressed, NO_MESSAGE_SENT, CLOSED, -1 if Dialog cancelled
```

Comment:
Is CLOSED == -1 ? If so, there seems to be no difference between cancelled and closed.
Maybe it doesn't matter, but if so, maybe it's better to use CLOSED than -1 ?

Reply:
Agreed, makes sense to use the local variable CLOSED